render-ws-java-client/src/test/java/org/janelia/render/client/AlignmentModelTest.java

```diff
@@ -100,6 +106,34 @@ public void interpolatingTheSameModelYieldsModel() {
 		assertArrayEquals(expected, actual, delta);
 	}
 
+	@Test
+	public void fittingYieldsRightResult() {
+		final InterpolatedAffineModel2D<AffineModel2D, TranslationModel2D> interpolated = new InterpolatedAffineModel2D<>(affine.copy(), translation.copy(), 0.5);
+		final AlignmentModel model = AlignmentModel.configure()
+				.addModel("affine", affine)
+				.addModel("translation", translation).build();
+		model.setWeights(Map.of("affine", 1.0, "translation", 1.0));
+
+		final List<PointMatch> matches = new ArrayList<>();
+		matches.add(new PointMatch(new Point(new double[] { 0, 0 }), new Point(new double[] { 0, 0.1 })));
+		matches.add(new PointMatch(new Point(new double[] { 0, 1 }), new Point(new double[] { 0.1, 1 })));
+		matches.add(new PointMatch(new Point(new double[] { 1, 0 }), new Point(new double[] { 1, 0 })));
+		matches.add(new PointMatch(new Point(new double[] { 1, 1 }), new Point(new double[] { 1, 1.1 })));
+		matches.add(new PointMatch(new Point(new double[] { 0.5, 0.5 }), new Point(new double[] { 0.5, 0.5 })));
+
+		try {
+			interpolated.fit(matches);
+			model.fit(matches);
+		} catch (final NotEnoughDataPointsException | IllDefinedDataPointsException e) {
+			fail(e.getMessage());
+		}
```

Comment:
Add throws clause to the test method signature and remove the try/catch statements.  This reduces test code, but even better you'll get the exception message along with a stack-trace from JUnit.